server/src/storage/segment.rs

```diff
@@ -132,18 +140,16 @@ pub struct SpilledSegment {
 }
 
 impl SpilledSegment {
-    /// Retrieve the data from the segment builder, save it to a Parquet file, and save the path.
-    pub fn new(key: String, mut segment_builder: SegmentBuilder) -> Self {
+    /// Save the data in `batch` to a Parquet file, and return a spilled segment with the path.
```

Comment:
Maybe replace "Save" with "Spil" to be consistent with `SpilledSegment` and `spill_to_parquet()`.

Reply:
Fixed.